pkgs/development/libraries/qt-4.x/4.8/default.nix

```diff
@@ -4,7 +4,7 @@
 , libmng, which, mesaSupported, mesa, mesa_glu, openssl, dbus, cups, pkgconfig
 , libtiff, glib, icu, mysql, postgresql, sqlite, perl, coreutils, libXi
 , buildMultimedia ? stdenv.isLinux, alsaLib, gstreamer, gst_plugins_base
-, buildWebkit ? stdenv.isLinux
+, buildWebkit ? true
```

Comment:
Why is this now set to `true`? Seems like a change for any library dependent on qt-4.8. Why not `buildWebkit ? stdenv.isLinux | stdenv.isDarwin`?